src/components/RoundScore/RoundScore.js

```diff
@@ -0,0 +1,148 @@
+import { h } from 'preact';
+import { formatScoreTime } from 'common/formatters';
+import cn from 'classnames';
+import Icon from 'react-fontawesome';
+import { holyWhite, winningGreen, superLightGreen, secondaryActionColor, partialCheater } from 'styles/colour-palette'
+
+const Solution = {
+    CORRECT: 'correct',
+    PARTIAL: 'partial'
+};
+
+function generateEntries(score) {
+    return score.map((entry) => {
+        const rootClasses = {
+            'score-entry': true,
+            '-solved': entry.correct === Solution.CORRECT,
+            '-partial': entry.correct === Solution.PARTIAL,
```

Comment:
remind me when we have this `PARTIAL` solution state?

Reply:
No idea, taken from _qd) But I can imagine a selector which selects the majority but not all of the required nodes.

Reply:
AFAIK we don't have partial solved state in `cssqd`. I guess `partial` state is when _qd solution passes public test, but fails with hidden, but we don't have hidden tests in `cssqd`. cc @kirilknysh

Reply:
Correct, not really valid for `cssqd`. For now - no real use cases for partial solutions here.

Reply:
Yes, this can be postponed until we're done with the CSSQD revamp and then do a (simpler) UI upgrade for _QD (that's what we agreed on, right?)

Reply:
removed